recipes/sphinxcontrib-towncrier/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sphinxcontrib-towncrier" %}
+{% set version = "0.4.0a0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sphinxcontrib-towncrier-{{ version }}.tar.gz
+  sha256: d9b1513fc07781432dd3a0b2ca797cfe0e99e9b5bc5e5c8bf112d5d142afb6dc
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools >=40.6.0
+    - setuptools-scm >=3.5
+    - setuptools_scm_git_archive >=1.1
+    - pip
+  run:
+    - python
+    - importlib-metadata >=4  # [py<38]
+    - sphinx
+    - towncrier >=19.2
+
+test:
+  imports:
+    - sphinxcontrib
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sphinx-contrib/sphinxcontrib-towncrier
+  summary: An RST directive for injecting a Towncrier-generated changelog draft containing fragments for the unreleased (next) project version
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - larsoner
```

Comment:
cc @webknjaz FYI I'm adding a conda-forge recipe for sphinxcontrib-towncrier so I can include it in a custom Anaconda installer. Let me know if you'd like me to add you as a maintainer as well!

Reply:
Sure, why not. Although, I'm not a part of this ecosystem so not sure if I'd be helpful.